packages/diagrams/backend/sirius-components-diagrams/src/main/java/org/eclipse/sirius/components/diagrams/Edge.java

```diff
@@ -289,6 +319,8 @@ public Edge build() {
             edge.endLabel = this.endLabel;
             edge.sourceId = Objects.requireNonNull(this.sourceId);
             edge.targetId = Objects.requireNonNull(this.targetId);
+            edge.modifiers = Objects.requireNonNull(this.modifiers);
+            edge.state = this.state;
```

Comment:
The state shouldn't be nullable

Reply:
The state cannot be null, it has a default value:
https://github.com/eclipse-sirius/sirius-components/blob/b19d7ba8c5538ac98dbc785ee8f0aacefa353ce2/packages/diagrams/backend/sirius-components-diagrams/src/main/java/org/eclipse/sirius/components/diagrams/Edge.java#L195
and the setter cannot accept null value:
https://github.com/eclipse-sirius/sirius-components/blob/b19d7ba8c5538ac98dbc785ee8f0aacefa353ce2/packages/diagrams/backend/sirius-components-diagrams/src/main/java/org/eclipse/sirius/components/diagrams/Edge.java#L284-L287
Do you want me to add the requireNonNull for readability reason?